x/evm/types/types.go

```diff
@@ -588,25 +588,25 @@ func CreateApproveContractCallWithMintCommand(
 }
 
 // decodeApproveContractCallWithMintParams unpacks the parameters of a approve contract call with mint command
-func decodeApproveContractCallWithMintParams(bz []byte) (string, string, common.Address, common.Hash, string, *big.Int, error) {
+func decodeApproveContractCallWithMintParams(bz []byte) (string, string, common.Address, common.Hash, string, *big.Int, common.Hash, *big.Int, error) {
```

Comment:
in the future, could we add a struct for the decoded params? The returns are getting ridiculous